server/routes/probationPractitionerReferrals/probationPractitionerReferralsController.ts

```diff
@@ -59,7 +60,19 @@ export default class ProbationPractitionerReferralsController {
     const { accessToken } = res.locals.user.token
 
     const existingDraftReferrals = await this.interventionsService.getDraftReferralsForUserToken(accessToken)
-    const presenter = new FindStartPresenter(existingDraftReferrals)
+
+    const structuredInterventionsFileDownloadPaths = {
+      xlsx: 'assets/downloads/Structured interventions list v1_3_3.xlsx',
+      pdf: 'assets/downloads/Structured interventions list v1_3_3.pdf',
+    }
+
+    const downloadFileSize = await FileUtils.fileSizeInKB(structuredInterventionsFileDownloadPaths.xlsx)
```

Comment:
I wonder what these relative paths will be resolved against here. Probably the process's current working directory. I think it would be good to be explicit here, I'll take a look at how these paths get configured for the assets.

Reply:
Oh, looks like they resolve relative to `process.cwd()` there, too. OK, will leave it.